examples/s3.tfnot

```diff
@@ -0,0 +1,149 @@
+module "terraform_pipeline" {
+  source  = "StratusGrid/multiaccount-pipeline/aws"
+  version = "~> #Relevant Version"
+
+  create                             = true
+  name                               = "${var.name_prefix}-utils"
+  codebuild_iam_policy               = local.terraform_pipeline_codebuild_policy
+  cb_env_compute_type                = "BUILD_GENERAL1_SMALL"
+  cb_env_image                       = "aws/codebuild/standard:5.0"
+  cb_env_type                        = "LINUX_CONTAINER"
+  cb_tf_version                      = var.terraform_version
+  cb_env_name                        = var.env_name
+  
+  # This is part of an or statement of code star or s3, this section is meant for if your artifacts are local and not in GIT. Use whitespace to emulate nulls, they must still be defined.
+  cp_resource_bucket_arn             = "arn:aws:s3:::bucket_name"
+  cp_resource_bucket_name            = "bucket_name"
+  cp_resource_bucket_key_name        = "codedump.zip" #See here for more data/examples https://docs.aws.amazon.com/codepipeline/latest/userguide/reference-pipeline-structure.html#action-requirements
+  cp_source_poll_for_changes         = true
+
+  # This is part of an or statement of code star or s3, this section is meant for if your artifacts are in GIT and not local. Use whitespace to emulate nulls, they must still be defined.
+  cp_source_owner                    = "myorg" # (GitHub/BitBucket Org) - (Organization Name/Project Name)
+  cp_source_repo                     = "myrepo" # Repository Name
+  cp_source_branch                   = "main" #Branch
+  cb_env_image_pull_credentials_type = "CODEBUILD"
+  cp_source_codestar_connection_arn  = ""
+  source_control                     = "GitHub" #GitHub or BitBucket
+
+  # This is used to enable slack notifications for codebuild statuses as well as codepipeline manual approval via AWS chatbot service 
+  slack_notification_for_approval    = true
+  slack_workspace_id                 = ""
+  slack_channel_id                   = ""
+  
+  # Each environment but be in the order, we prefix this list since the map will sort alphabetically and we can not change that.
+  # We make an assumption that the environment name matches the environment name in the TF init and apply directories.
+  cb_accounts_map = {
+    "dev" = {
+      account_id = "0012345678901"
+      iam_role   = "iam-cicd"
+      manual_approval = false
+      order = 1
+    }
+    "stg" = {
+      account_id = "123456789012"
+      iam_role   = "iam-cicd"
+      manual_approval = true
+      order = 2
+    }
+    "prd" = {
+      account_id = "234567890123"
+      iam_role   = "iam-cicd"
+      manual_approval = true
+      order = 3
+    }
+  }
+}
+
+locals {
```

Comment:
Convert policy to data source

Reply:
Dang you and you using my own logic against me.

Reply:
Fixed under https://www.youtube.com/watch?v=ObisQoGN6Cc